sale_restricted_qty/__manifest__.py

```diff
@@ -6,8 +6,7 @@
     "version": "14.0.1.1.2",
     "category": "Sales Management",
     "author": "Akretion, Odoo Community Association (OCA)",
-    "contributors": ["Ashish Hirpara"],
-    "maintainers": ["ashishhirapara"],
+    "maintainers": ["alexey-pelykh"],
```

Comment:
Please don't remove the current one. Add yourself.

Reply:
I think that one is unresponsive, seeing the previous PRs, so why keeping it?

Reply:
That's my thought as well, there's no issue re-adding if anything changes. I would assume based on activity a lot of module maintainers (me including on some modules) should've been removed.

Reply:
Yes but sometimes, user is simply out of order but want to just maintain one module.

Reply:
Well, the person will have some time to object while 5 days cooldown time is there.